python/lsst/ts/scriptqueue/block_info.py

```diff
@@ -0,0 +1,151 @@
+# This file is part of ts_scriptqueue.
+#
+# Developed for the LSST Telescope and Site Systems.
```

Comment:
Please update the license headers of all Python files to refer to Vera C. Rubin Observatory.